src/style.css

```diff
@@ -0,0 +1,1252 @@
+/* @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap'); */
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,700;1,400;1,500&display=swap'); */
+/* @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,500;0,700;1,500&display=swap'); */
+@import url('https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;0,600;0,700;1,600&display=swap');
+
+:root {
+    font-size: 16px;
+    --font-family: 'Nunito', sans-serif;
+
+    --primary: #9333EA;
+    --primary-dark: #6B21A8;
+    --primary-100: #F3E8FF;
+    --primary-400: #C084FC;
+    --grey-100: #F3F4F6;
+    --grey-200: #E5E7EB;
+    --grey-300: #D1D5DB;
+    --grey-400: #9CA3AF;
+    --grey-500: #6B7280;
+    --grey-600: #4B5563;
+    --red-100: #FEE2E2;
+    --red-600: #DC2626;
+    --red-800: #991B1B;
+    --green-100: #D1FAE5;
+    --green-600: #059669;
+    --green-800: #065F46;
+    --yellow--100: #FEF3C7;
+    --yellow-400: #FBBF24;
+    --yellow-700: #B45309;
+    --text-primary: #111827;
+    --text-secondary: #F9FAFB;
+    --success: #38a169;
+    --failure: #e53e3e;
+
+    /* Spacing */
+    --space-0: 0.25rem;
+    --space-1: 0.5rem;
+    --space-2: 1rem;
+    --space-3: 1.5rem;
+    --space-4: 2rem;
+    --space-5: 2.5rem;
+    --space-6: 3rem;
+    --space-7: 3.5rem;
+    --space-8: 4rem;
+
+    /* Font Size */
+    --size-1: 0.5rem;
+    --size-2: 1rem;
+    --size-3: 1.5rem;
+    --size-4: 2rem;
+    --size-5: 2.5rem;
+    --size-6: 3rem;
+    --size-7: 3.5rem;
+    --size-8: 4rem;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: var(--font-family);
+    z-index: 0;
+}
+
+.pos-rel {
+    position: relative;
+}
+
+.hide-scroll {
+    overflow: hidden;
+}
+
+/* Background Color Utility */
```

Comment:
i would try to use !important sparingly as it might create issue later as library scales and more components are added but this looks good for now.  i liked that you used CSS variables to have consistency across the entire project.